File: Tareas/14/Código RUBY/tarea14.rb
=begin
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
=end

print("Ingrese el numero de terminos: ")
n = gets().to_i

serie = ""
i = 1
for i in(1...n)
	a = 1.0/i

	serie = serie + a.to_s + ", "
end

a = 1.0/n
serie = serie + a.to_s + "."

print(serie)

File: Tareas/14/Código JAVA/tarea14.java
/*	
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
*/

import javax.swing.JOptionPane;
public class tarea14{
	public static void main (String[] args){
		int i, n;
		double a;
		String entrada;
		String serie = "";

		entrada = JOptionPane.showInputDialog("Ingrese el numero de terminos: ");
		n = Integer.parseInt(entrada);

		for(i=1; i<n; i++){
			a = 1.0/i;
			serie = serie + a + ", ";
		}
		a = 1.0/n;
		serie = serie + a + ".";
		JOptionPane.showMessageDialog(null,serie);
	} //Fin de función main
} //Fin de clase tarea14